supabase/rollback/20260829030000_candidate_roster_publication_integrity.rollback.sql

```
@@ -0,0 +1,143 @@
+      WHERE source <> 'TSE consulta_cand e consulta_cand_complementar 2026; snapshot 27/08/2026'
+    ),
+    ultima_atualizacao=now()
+WHERE slug IN (
+  'andre-luis','carlos-machado','dario-barbosa','elisson-ferreira','eudo-raffael',
+  'francisco-jurity','huggo-leonardo','kiko-caputo','leonardo-avalanche','renato-gomes',
+  'rico-pinheiro','roberio-paulino','well-macedo','witer-naves','yuri-ezequiel'
+);
+
+-- Restaura somente os campos que estavam vazios antes da forward. As listas
+-- são derivadas do artefato de remediação versionado e preservam dados prévios.
+UPDATE public.candidatos SET genero=NULL WHERE slug IN (
```

SQLFluff (4.3.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)